test/jdk/javax/swing/JMenu/4213634/bug4213634.java

```diff
@@ -77,33 +71,30 @@ public  void createAndShowGUI() {
         frame.getContentPane().add("Center", ta);
```

Comment:
The text area seems redundant, especially the text inside which looks like an [Easter egg](https://en.wikipedia.org/wiki/Easter_egg_(media)).